bin/release.py

```diff
@@ -159,35 +170,19 @@ def prepare_release_artifact(work_dir, build_job_number, version, arch, waf):
     module_debug_path = work_dir / "ngx_http_datadog_module.so.debug"
     download_file(module_url_dbg, module_debug_path)
 
-    waf_suffix = "-appsec" if waf else ""
-
+    # Package and sign .so
     tarball_path = (
         work_dir /
         f"ngx_http_datadog_module{waf_suffix}-{arch}-{version}.so.tgz")
-    command = [tar_exe, "-czf", tarball_path, "-C", work_dir, module_path.name]
-    run(command, check=True)
-
-    command = [gpg_exe, "--armor", "--detach-sign", tarball_path]
-    run(command, check=True)
+    package(module_path, out=tarball_path)
+    sign_package(tarball_path)
 
+    # Package and sign .so.debug
     debug_tarball_path = (
         work_dir /
         f"ngx_http_datadog_module{waf_suffix}-{arch}-{version}.so.debug.tgz")
-    command = [
-        tar_exe,
-        "-czf",
-        debug_tarball_path,
-        "-C",
-        work_dir,
-        module_debug_path.name,
-    ]
-    run(command, check=True)
-
-    command = [gpg_exe, "--armor", "--detach-sign", debug_tarball_path]
-    run(command, check=True)
-
-    module_path.unlink()
-    module_debug_path.unlink()
```

Comment:
Is there any reason to not delete the files now that did apply before this change? Or were these removed because they are not necessary?